content/en/docs/concepts/cluster-administration/priority-and-fairness.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Priority and Fairness in Handling Requests
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
```

Comment:
I think for a concept page it's appropriate to outline both the existing and proposed behavior.
(if that sounds like a good idea, I'll offer to help with documenting the current behavior, and that could potentially land in `master` even before the 1.16 release).

Reply:
Are you suggesting one page that documents both, or one page that documents the old and one that documents the new?

Reply:
Either's fine by me.